frontend/src/pages/store/[store].svelte

```diff
@@ -479,28 +520,66 @@
 				>
 			</div>
 		{/if}
-		{#if error || !items || items.length == 0}
-			<h2 class="text-center">
+		<div class="flex space-x-2 mb-4">
```

Comment:
Right now these feel really close together, you might want to make the spacing bigger to something like `space-x-4`